docs/application/native/guides/ui/dali/videoview.md

```diff
@@ -108,6 +110,16 @@ newMap.Insert( "volumeRight", 0.5f );
 mView.SetProperty( VideoView::Property::VOLUME, newMap );
 ```
 
+How to use DISPLAY_MODE.
+
+there are enum of display modes: LETTER_BOX, ORIGIN_SIZE, FULL_SCREEN, CROPPED_FULL, ORIGIN_OR_LETTER, DST_ROI (See [multimedia display](https://developer.tizen.org/dev-guide/3.0.0/org.tizen.native.mobile.apireference/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html))
```

Comment:
Hello, @JoogabYun 

How about changing the update by you in below form?
The re-written below needs your check. If you think this modification is acceptable, please update the PR.

---

To change the display mode, use the `PLAYER_DISPLAY_MODE` enums (in [mobile](../../../api/mobile/latest/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html#gaafcf6f7487d8cb376e1851a6e1253f3a) and [wearable](../../../api/wearable/latest/group__CAPI__MEDIA__PLAYER__DISPLAY__MODULE.html#gaafcf6f7487d8cb376e1851a6e1253f3a) applications) .
 - `LETTER_BOX`
 - `ORIGIN_SIZE`
 - `FULL_SCREEN`
 - `CROPPED_FULL`
 - `ORIGIN_OR_LETTER`
 - `DST_ROI`
```
mView.SetProperty( VideoView::Property::DISPLAY_MODE, VideoView::DisplayMode::LETTER_BOX );
```

---

Hello, @annie-samsung 
If, JoogabYun agrees with this modification, in that case please review the sentence of above.
If he doesn't accept it, then please review the original update by JoogabYun.